src/__test__/Rockets.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Rockets from '../components/Rockets';
+// import rocketReducer from '../redux/rockets/rockets';
+
```

Comment:
Kindly consider removing unused code in the code base, removing unused code makes your work easy to follow and more professional